vortex-serde/src/io/dispatcher/mod.rs

```diff
@@ -0,0 +1,77 @@
+#[cfg(feature = "tokio")]
```

Comment:
this module is heavily inspired by the `compio-dispatcher` crate: https://github.com/compio-rs/compio/blob/master/compio-dispatcher/src/lib.rs